test/test_external.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from otk.external import convert
+
+
+@pytest.mark.parametrize("text,exe,args", [
```

Comment:
Is the json handling tested elsewhere or should there be a followup with some more testing (afaict `external.py:call()` is currently untested(?)).

Reply:
Some more testing should eventually happen :)